src/afl-fuzz-init.c

```diff
@@ -914,6 +914,11 @@ void perform_dry_run(afl_state_t *afl) {
 
     res = calibrate_case(afl, q, use_mem, 0, 1);
 
+    /* For AFLFast schedules we update the queue entry */
+    if (likely(q->exec_cksum)) {
```

Comment:
this should be:
`if (unlikely(afl->schedule >= FAST && afl->schedule <= RARE) && likely(q->exec_cksum)) {`